### DISCUSSION TITLE
Welcome to rlxos Discussions

### DISCUSSION THREAD

**Discussion by itsManjeet** [itsManjeet/avyos#21] [Announcements]
- Ask any questions related to rlxos.
- Get help with installation, setup, and troubleshooting.
- Discuss command-line usage and system administration.
- Connect with fellow software developers and discuss rlxos specific tools and programming languages.
- Share your experiences, tips, and tricks with the community.
- Foster a positive and inclusive environment.
- Be respectful towards others and encourage collaboration.
- Let's build a supportive community where everyone feels welcome.

**Comment by ghost** [comment]
Finally!

**Comment by adams-j** [comment]
Hello, I have installed rlxos and I really appreciate your work/design/direction of this distribution. 

I am not able to find the documentation that is located in a root docs/ directory, I am specifically looking to connect with my network printer and looking for guidance on the process.  

Can you guide me to locate the documentation?

**Reply by itsManjeet** [reply]
Docs are hosted at https://docs.rlxos.dev, but docs are limited to rlxos core system only, you can follow https://wiki.archlinux.org/title/CUPS arch wiki guide for that query, CUPS is already preconfigured in rlxos. just follow the steps to connect with your printer.

**Reply by adams-j** [reply]
Thanks for the quick reply, looking into distrobox, I noticed cups was preconfigured also

**Reply by itsManjeet** [reply]
Look like firefox is not opening URL as expected, try url http://localhost:631 otherwise it is expected to work from Application Menu -> System -> Manage Printing

**Comment by adams-j** [comment]
Another hurdle I have come across is getting a VPN to work.  I installed debian via distrobox, installed the vpn in debian, exported the vpn app. This method did not work.  

Currently I am thinking about installing wireguard on debian, but I did find a vpn connection already in rlxos, is there a way to install a vpn plugin that works with that? 

I figured you guys have a solution, thanks for the help.

**Reply by itsManjeet** [reply]
VPN work on the IP, and might not have a straight way to manipulate that from inside a container,
You can do, Select the Network Applet from the bottom right of the panel -> VPN Connections -> Add VPN Connection

But wait, NetworkManager-openvpn and other module is missing in the system.

Creating a issue here. https://github.com/itsManjeet/rlxos/issues/92

Thanks for report.

**Reply by adams-j** [reply]
I saw OpenVPN NetworkManager plugin added on the last update, thanks
